### ISSUE TITLE
Check if image file exists in the react native project path

### ISSUE THREAD

**Issue by beisert1** [itinance/react-native-fs#494]
I need to be able to check if an image file exists in the project path. Example "../assets/images/picture.jpg"

I know how to check if it exists in the device storage and how to download the file from server to device storage but I first need to check if the image is packaged into the project.

**Comment by itinance** [comment]
is .../assets a folder within the javascript-space? if so: no chances and beyond the scope of this library.

**Comment by beisert1** [comment]
@itinance Yes /assets folder is under my src folder. I understand, thank you for the quick reply.

**Comment by jeroen-van-dijk** [comment]
@beisert1 Where you able to verify the existence of the file from the project folder somehow?
